internal/types/document.go

```diff
@@ -254,6 +254,21 @@ func (d *Document) Set(key string, value any) error {
 	return nil
 }
 
+func (d *Document) Empty() bool {
```

Comment:
We use `d.Len() == 0` in some other places, let's continue using it

Reply:
applied